src/routes/integrations/slack.ts

```diff
@@ -187,7 +192,10 @@ export default async function (fastify: FastifyInstance): Promise<void> {
         );
       }
 
-      return redirectResponse({ res, path: redirectPath });
+      return redirectResponse({
+        res,
+        path: `${redirectPath}&iid=${integrationId}`,
```

Comment:
Let's use URL to assemble because we can't know if `redirectPath` already contains any search params or not

Reply:
Good point :) I'll address